### PR TITLE
feat: create OS Information guide #1484

### PR DESCRIPTION
**Pull request by vasfvitor**
#### What kind of changes does this PR include?
- New content

#### Description
I'm putting this out but I'm not sure what the Rust code, or even the JS should included, I mean, how much detail or step. Right now the Rust code is minimal, so is the JS, it's almost like the js-api reference. For comparison in the clipboard guide #1622, I've included three steps to use the Rust code, with the end goal of invoking in frontend.

edit: ok, I'm readin https://diataxis.fr/

### CODE REVIEW

**Comment by lorenzolewis** [comment]
To your point on this being a JS/Rust API reference I agree. I think that instead of listing out each different function, we could instead just have on function (such as `platform`) and then add a sentence saying additional functions as available with links to the API docs for each one.

**Comment by vasfvitor** [comment]
Is that how you imagined?

File: src/content/docs/features/os-info.mdx [hunk 1]
```diff
@@ -3,11 +3,105 @@ title: OS Information
...
+        pnpm="pnpm add @tauri-apps/plugin-os"
+    />
+
+    </TabItem>
+
+</Tabs>
+
+## Usage
+
+With this plugin you can query multiple information from current operational system, here is how you can utilize the OS Information plugin:
```
Comment (lorenzolewis) [inline src/content/docs/features/os-info.mdx]:
```suggestion
With this plugin you can query multiple information from current operational system. See all available functions in the [JavaScript API](/2/reference/js/os/) or [Rust API](https://docs.rs/tauri-plugin-os/2.0.0-alpha.4/tauri_plugin_os/) references.
```

File: src/content/docs/features/os-info.mdx [hunk 2]
```diff
@@ -3,11 +3,105 @@ title: OS Information
...
+```rust
+let platform = tauri_plugin_os::platform();
+println!("Platform: {}", platform);
+// Prints "windows" to the terminal
+```
+
+</TabItem>
+</Tabs>
+
+See all available functions in the [JavaScript API](/2/reference/js/os/) or [Rust API](https://docs.rs/tauri-plugin-os/2.0.0-alpha.4/tauri_plugin_os/) references.
```
Comment (lorenzolewis) [inline src/content/docs/features/os-info.mdx]:
```suggestion
```

**Review by lorenzolewis — CHANGES_REQUESTED** [review]
Would just move this link to be further up at the top so the reader can more easily see it.

**Review by lorenzolewis — APPROVED** [review]
Awesome! And good catch with the docs.rs link

**Pull request merged**